Fmaraev/Hometask_6/Task_4/Task_4.py

```diff
@@ -0,0 +1,46 @@
+with open('ratings.list', encoding="ISO-8859-1") as rating:
+    films_list = []
+    lines = [el for el in range(28, 278)]
+    i = 0
+    for line in rating:
+        if i in lines:
+            films_list.append(line.strip())
+            i += 1
+        else:
+            i += 1
+
+new_films_list = [el.split('  ') for el in films_list]
+
+movie_titles = [el[-1] for el in new_films_list]
+
+movie_ratings = [el[-2].strip() for el in new_films_list]
+movie_ratings = list(map(float, movie_ratings))
+print(movie_ratings)
+
+some_list = [el.split() for el in movie_titles]
+movie_release = [el[-1].strip('()/I') for el in some_list]
+movie_release = list(map(int, movie_release))
+print(movie_release)
+
+
+with open('top250_movies.txt', 'w') as top:
+    content = '\n'.join(movie_titles)
+    top.write(content)
+
+with open('ratings.txt', 'w') as ratings:
+    for el in movie_ratings:
+        toprint = ''
+        times = el
+        while times > 7:
+            toprint += '$'
+            times = times - 0.05
+        ratings.write(toprint + '\n')
+
+with open('years.txt', 'w') as years:
+    for el in movie_release:
+        toprint = ''
+        times = el
+        while times > 1900:
```

Comment:
it should be like `1999 ****` that means that 4 from TOP 250 movies were produced in 2000